spec/src/main/asciidoc/changelog.adoc

```diff
@@ -30,6 +30,7 @@ Changes marked with icon:bolt[role="red"] are breaking changes relative to previ
 === API/SPI Changes
 ** `MetricRegistry` changed from `abstract class` to `interface`
 ** icon:bolt[role="red"] Removed everything related to reusability from the API code
+** icon:bolt[role="red"] CDI producers annotated with `@Metric` no longer trigger metric registration
```

Comment:
You need to create the section and call out the Breaking changes. See [here](https://wiki.eclipse.org/MicroProfile/ReleaseNotes) for more info.

Reply:
Restructured it a bit now.